query-engine/core/src/query_graph_builder/extractors/query_arguments.rs

```diff
@@ -79,8 +81,8 @@ fn extract_order_by(model: &ModelRef, value: ParsedInputValue) -> QueryGraphBuil
                         let field = model.fields().find_from_scalar(&field_name)?;
                         let value: PrismaValue = sort_order.try_into()?;
                         let sort_order = match value.into_string().unwrap().to_lowercase().as_str() {
-                            "asc" => SortOrder::Ascending,
-                            "desc" => SortOrder::Descending,
+                            filters::ASC => SortOrder::Ascending,
```

Comment:
`asc` / `desc` probably doesn't belong into `filter`, it's more of an arg or separate, like `ordering`.